ngraph/test/op_eval/atan.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+#include <vector>
+
+#include "gtest/gtest.h"
+
+#include "ngraph/op/atan.hpp"
+#include "ngraph/runtime/host_tensor.hpp"
+#include "ngraph/validation_util.hpp"
+#include "runtime/backend.hpp"
+#include "util/test_tools.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+TEST(op_eval, atan)
```

Comment:
Please use backend tests instead. Thread here: https://github.com/openvinotoolkit/openvino/pull/6361#discussion_r660482249

Reply:
per #6361, drop the op_eval test, been covered in backend test